src/main/java/org/sagebionetworks/bridge/exporter/util/BridgeExporterUtil.java

```diff
@@ -36,29 +34,11 @@
     public static final String CONFIG_KEY_TIME_ZONE_NAME = "time.zone.name";
     public static final String CONFIG_KEY_RECORD_ID_OVERRIDE_BUCKET = "record.id.override.bucket";
     public static final String CONFIG_KEY_SQS_QUEUE_URL = "exporter.request.sqs.queue.url";
-
+    public static final String DEFAULT_TABLE_NAME = "Default Health Data Record Table";
```

Comment:
"Schemaless Health Data Record Table" or "No Schema Health Data Record Table" ?

Reply:
I feel like "Default" better conveys what the table is than "Schemaless". The idea is that, going forward, Schemaless is the norm rather than the exception. So in this case, it really is the default.